_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -528,17 +526,57 @@ ggplot(data = gapminder_1997) +
 > ```{r plotErrors, eval = FALSE}
```

Comment:
Above here, should change errors to bugs since the 1st bug doesn't return an error? Doesn't really matter to me, but just had that though